### ISSUE TITLE
Usersnap Feedback - Slight problem (or maybe not):  Because the brief description appears abov[...]

### ISSUE THREAD

**Issue by ascott** [participedia/usersnaps#467]
**Sender**: scottowenfletcher@gmail.com
**Comment**: Slight problem (or maybe not): 
Because the brief description appears above the 'Problems and Purpose' heading, it will also come before the editing prompts Jaskiran and I use (see this case for example). I think we may need to design an admin tool to insert an editing prompt before the brief description. see here for a list of all editing prompts used (2nd page): https://docs.google.com/document/d/1FUGo2__6LrOn2xOEnU1olqOQXVOa8Wm-iLwqyakc2fo/edit?usp=sharing
[Open #172 in Usersnap Dashboard](https://usersnap.com/a/#/participedia-dev/p/participedia-201-a37e4cf6/172)

<a href='https://usersnappublic.s3.amazonaws.com/2019-05-15/18-22/e114e1f1-14d2-41a4-8c1c-d1412bd3d252.png'>![Usersnap Feedback - Slight problem (or maybe not):  Because the brief description appears abov(...)](https://usersnappublic.s3.amazonaws.com/2019-05-15/18-22/55bffaf2-1c48-4ec6-8452-88c8ce8120e2.jpg)</a>
<a href='https://usersnappublic.s3.amazonaws.com/2019-05-15/18-22/e114e1f1-14d2-41a4-8c1c-d1412bd3d252.png'>Download original image</a>

**Browser**: Firefox 66 (Windows 10)
**Referer**: [https://ppedia-stage.herokuapp.com/case/1394](https://ppedia-stage.herokuapp.com/case/1394)
**Screen size**: 1920 x 1080 **Browser size**: 1920 x 877

Powered by [usersnap.com](https://usersnap.com/?utm_source=github_entry&utm_medium=web&utm_campaign=product)

**Comment by jesicarson** [comment]
@scottofletcher you could put your prompt in the brief description field if you want it to show up at the top. Also, brief descriptions are short enough that the prompt wont be too far down the page.. I think it might be ok as is. But lets see how they start shaping up.

**Comment by scottofletcher** [comment]
@jesicarson yes, that could work but a lot of stubs have a brief description and putting the editing prompt in the same box would reduce the number of characters. I think what I might do is put a space before and after the editing prompt. I'll try that out when the site is back up and we can assess how it looks

**Comment by jesicarson** [comment]
reference: https://github.com/participedia/usersnaps/issues/468
